### ISSUE TITLE
Have an ability to have draft posts

### ISSUE THREAD

**Issue by nisrulz** [zzossig/hugo-theme-zzo#108]
The theme publishes everything. Even the ones set to drafts.

Was hoping this was respected: https://simoncann.com/wotl/hugo-draft-future/#drafts

**Comment by VictorZakharov** [comment]
No it's not. It will only publish if you are running a dev server. It won't go to production. Try it.

**Comment by zzossig** [comment]
I suspect you might run Hugo by typing this
```
Hugo server -D
```

`-D` flag means `buildDrafts`. So just try this.

```
Hugo server
```

**Comment by nisrulz** [comment]
I was using `hugo server -D` for dev env, so I understand that part. 

But somehow  when I did `hugo -d ../some_folder/` it published one of my draft posts.

I retried it again and this time it didn't. Not sure what I did wrong. 

Closing as obviously it works as expected. I could have done something wrong, since the theme doesn't change what hugo supports by default.

Thank you for your comments though. Appreciated.
